packages/design-system-scripts/gulp/common/constants.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 
 const CORE_SOURCE_PACKAGE = '@cmsgov/design-system';
-const CORE_DOCS_PACKAGE = '@cmsgov/design-system-docs';
+const CORE_DOCS_PACKAGE = '@cmsgov/cms-design-system-docs';
```

Comment:
Curious what this docs package is? Thought we were deleting the design-system-docs and this looks like a new/different docs package

Reply:
It's just the current name of the ./docs in it's package.json, this line can likely be removed, i'll take a look.